scripts/sct_deepseg.py

```diff
@@ -145,4 +145,5 @@ def main():
 
 
 if __name__ == '__main__':
+    init_sct()
```

Comment:
Observation: when doing:
```
init_sct(update=True)
```
--> it solves #2993 , ie no duplication of the output / fsleyes command --> the issue may come from [here](https://github.com/neuropoly/spinalcordtoolbox/blob/5a23e7df5d9b99a661d8a74b684330fdcfe9b8d0/spinalcordtoolbox/utils/sys.py#L67).